docs/reference/dev.rst

```diff
@@ -18,7 +18,7 @@ Linux or macOS.  Windows is not currently supported.
 * C compiler (GCC or clang);
 * Rust compiler and Cargo 1.53 or later;
 * autotools;
-* Python 3.9 dev package;
+* Python 3.10 dev package;
```

Comment:
Oh, while here, can you add one more commit mentioning the requirement for yarn/node?

Reply:
That was already in the pending #3725, which I just went and merged